src/styles/blocks/header-top.scss

```diff
@@ -0,0 +1,33 @@
+.header-top {
+  padding-block: 24px;
```

Comment:
I don't think this approach is the most optimal. 
You are positioning children elements inside the container => parent should be responsible for positioning it's children. Instead of adding padding to `header-top` && `header__bottom` you should set padding to `header__content`.